assignments/homework/pcosta/497HW3.py

```diff
@@ -0,0 +1,30 @@
+from astropy.wcs import WCS
+from astropy.io import fits
+import matplotlib.pyplot as plt
+
+#WCS Notebook
+
+imagehdulist = fits.open('../../../astropy_notebooks/data/LMCDensFits1k.fits')
+plt.imshow(imagehdulist[0].data)
+gaiaHDUlist = fits.open('../../../astropy_notebooks/data/gaia_lmc_psc.fits')
+gaiaData = gaiaHDUlist[1].data
+wcs = WCS('../../../astropy_notebooks/data/LMCDensFits1k.fits')
+px, py = wcs.wcs_world2pix(gaiaData['ra'], gaiaData['dec'], 0)
```

Comment:
if you check the `wcs` of the previous line, it says the axes are in galactic coordinate frame. However, with this line you feed in icrs coordinates, and thus px and py are not overlapping with the image data in the figure. 
You could either: convert the ra-dec pair to l-b pairs of galactic coordinates and feed them as float arrays to `wcs_world2pix` or rather make a separate `SkyCoord` from the ra-dec columns and then use `wcs.world_to_pixel` with that SkyCoord as argument. I would recommend the latter as it would automatically take care of the frame transformation, but the former approach should work, too.